ee/vellum_ee/workflows/display/tests/workflow_serialization/generic_nodes/test_adornments_serialization.py

```diff
@@ -237,124 +237,82 @@ class StackedWorkflow(BaseWorkflow):
-                                },
-                                "attributes": [
-                                    {
-                                        "id": "ab2fbab0-e2a0-419b-b1ef-ce11ecf11e90",
-                                        "name": "on_error_code",
-                                        "value": {"type": "CONSTANT_VALUE", "value": {"type": "JSON", "value": None}},
-                                    }
-                                ],
-                            },
-                        ],
-                        "attributes": [],
-                        "outputs": [],
+                        {
+                            "id": "73a02e62-4535-4e1f-97b5-1264ca8b1d71",
+                            "name": "retry_on_condition",
+                            "value": {"type": "CONSTANT_VALUE", "value": {"type": "JSON", "value": None}},
+                        },
+                    ],
+                },
+                {
+                    "id": "3344083c-a32c-4a32-920b-0fb5093448fa",
+                    "label": "TryNode",
+                    "base": {
+                        "name": "TryNode",
+                        "module": ["vellum", "workflows", "nodes", "core", "try_node", "node"],
                     },
-                ],
-                "edges": [
-                    {
-                        "id": "e8bd50dd-37a0-49b0-8b7b-f1dd8eb478b9",
-                        "source_node_id": "c14c1c9b-a7a4-4d2c-84fb-c940cfb09525",
-                        "source_handle_id": "51a5eb25-af14-4bee-9ced-d2aa534ea8e9",
-                        "target_node_id": "074833b0-e142-4bbc-8dec-209a35e178a3",
-                        "target_handle_id": "6e4af17f-bbee-4777-b10d-af042cd6e16a",
-                        "type": "DEFAULT",
-                    }
-                ],
-                "display_data": {"viewport": {"x": 0.0, "y": 0.0, "zoom": 1.0}},
-                "definition": {
-                    "name": "StackedWorkflow",
-                    "module": [
-                        "vellum_ee",
-                        "workflows",
-                        "display",
-                        "tests",
-                        "workflow_serialization",
-                        "generic_nodes",
-                        "test_adornments_serialization",
+                    "attributes": [
+                        {
+                            "id": "ab2fbab0-e2a0-419b-b1ef-ce11ecf11e90",
+                            "name": "on_error_code",
+                            "value": {"type": "CONSTANT_VALUE", "value": {"type": "JSON", "value": None}},
+                        }
                     ],
                 },
-            },
-            "input_variables": [],
-            "state_variables": [],
-            "output_variables": [],
+            ],
+            "attributes": [],
+            "outputs": [],
         },
-        exec_config,
-        ignore_order=True,
```

Comment:
We should look to `ignore_order` entirely from this repo. We actually really care about order bc we need to ensure consistent snapshots/codegens